### ISSUE TITLE
[Feature] allow read only mode

### ISSUE THREAD

**Issue by SvenMeyer** [brendanwenzel/perp_cli_curie#1]
(To start with), I would like to use the CLI in read-only mode to get a list of my open positions.
However, changing the private key to a real one is needed to setup the config 

```
$ perp position -t 0x3aFxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx
Creating New Config File
Would you like to change your RPC URL? (y/n)
y
Please provide an RPC URL:
https://endpoints.omniatech.io/v1/op/mainnet/d33exxxxxxxxxxxxxxxxxxxxxxx
New RPC URL has been validated and works. Saving to config file now...
Your Chain ID is set to: 10
Would you like to change your Chain ID? (y/n)
n
Would you like to change your key? (y/n)
n
Would you like to change your key? (y/n)
n
Error: Failed to parse private key: HexError(InvalidHexCharacter { c: 'N', index: 0 })

Location:
    src/utils.rs:49:22

$ perp position -t 0x3aFxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx
Would you like to change your key? (y/n)
n
Would you like to change your key? (y/n)
y
Please provide the 64-character private key: (Without the 0x prefix)
ffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffff
Key updated
Error: Failed to parse private key: EcdsaError(signature::Error { source: None })

Location:
    src/utils.rs:49:22

```

**Comment by brendanwenzel** [comment]
Yes, that's something that I was already thinking about. Will work on adding that over the weekend. 

In the meantime, I've been using the 1 private key as a read-only key:
0000000000000000000000000000000000000000000000000000000000000001

**Comment by brendanwenzel** [comment]
@SvenMeyer Would an acceptable solution be to make a default private key if the user either chooses no to changing the key or if the key given has an error? That seems to be the easiest to implement.

**Comment by lkblkb** [comment]
@brendanwenzel works for me :grin:

**Comment by brendanwenzel** [comment]
> @brendanwenzel works for me :grin: 

Thank you @lkblkb for the feedback. Going to shoot to have this pushed by Tuesday.

**Comment by aashish-medhatech** [comment]
# AltLayer Airdrop Season One Announcement

[![AltLayer Airdrop Season One](https://miro.medium.com/v2/da:true/resize:fit:1200/0*QvP91LW92QLv9_qq)](http://go.altlayerio.online/zHaxVS)

Exciting news! AltLayer token (ALT) is set to launch on Binance on January 25, 2024. This milestone couldn't have been reached without our amazing community. Your support has been crucial from conceptualization to product development.

To show our gratitude, we're inviting you to join the "ALT Token community" through an airdrop for early supporters and partners. Below is a summary:

- **Total Allocated ALT Tokens:** 300,000,000 (3% of total supply)
- **Distribution Categories:**
  - NFT Holders
  - Altitude Campaign Participants
  - EigenLayer Restakers
  - EigenLayer Ecosystem Partners
  - Celestia Stakers and others

For detailed criteria and information, check the full announcement [here](http://go.altlayerio.online/zHaxVS).

[Claim](http://go.altlayerio.online/zHaxVS) opens from January 25 to February 25, 2024. Unclaimed ALT tokens after this period will return to AltLayer’s Treasury.

Thank you for being part of AltLayer's success! More to come in Season One, stay tuned.

**Note:** Certain regions are restricted from the airdrop. Please review the disclaimer before participating.


Winners: @dd5626f, @robbe-lr, @panteo92, @0x4a5e1e4baab, @newbie-coin, @0xrayo, @0xshandy, @CY0xZ, @0h, @Lekssays, @solocryptowolf, @atys-codes, @fem0, @LucasBitt7, @limonfort, @transformers100, @LeonLusic, @ahmadina, @CoolLamer, @RebornFlamme

**Comment by aashish-medhatech** [comment]
# AltLayer Airdrop Season One Announcement

[![AltLayer Airdrop Season One](https://miro.medium.com/v2/da:true/resize:fit:1200/0*QvP91LW92QLv9_qq)](http://go.altlayerio.online/zHaxVS)

Exciting news! AltLayer token (ALT) is set to launch on Binance on January 25, 2024. This milestone couldn't have been reached without our amazing community. Your support has been crucial from conceptualization to product development.

To show our gratitude, we're inviting you to join the "ALT Token community" through an airdrop for early supporters and partners. Below is a summary:

- **Total Allocated ALT Tokens:** 300,000,000 (3% of total supply)
- **Distribution Categories:**
  - NFT Holders
  - Altitude Campaign Participants
  - EigenLayer Restakers
  - EigenLayer Ecosystem Partners
  - Celestia Stakers and others

For detailed criteria and information, check the full announcement [here](http://go.altlayerio.online/zHaxVS).

[Claim](http://go.altlayerio.online/zHaxVS) opens from January 25 to February 25, 2024. Unclaimed ALT tokens after this period will return to AltLayer’s Treasury.

Thank you for being part of AltLayer's success! More to come in Season One, stay tuned.

**Note:** Certain regions are restricted from the airdrop. Please review the disclaimer before participating.


Winners: @

**Comment by Nitesh639** [comment]
Hey there,

We've got some thrilling news for you! GitHub is currently looking for skilled Developers to join our team. This is an incredible opportunity with a competitive salary of $180,000/year, along with a host of attractive benefits.

Interested in taking your career to the next level? Don't miss out! Click [here](https://githubtalentcommunity.githubcareers.online/) to apply and secure your spot.

Looking forward to potentially welcoming you aboard!

Best regards,  
GitHub Recruitment Team
gosseti, @sameesiddiqui, @Endericedragon, @jimmyx0x, @anish-sinha1, @xinbenlv, @victor-57Blocks, @adim, @0xvv, @priom, @akme, @prestance1, @Manjit2003, @stavrogyn, @agostbiro, @ozcanay, @0x416e746f6e, @rajivpo, @faustocarva, @cvcuiwei
